mono/metadata/class-init.h

```diff
@@ -10,6 +10,9 @@
 #include <mono/metadata/metadata.h>
 #include <mono/metadata/class-internals.h>
 
+gboolean
+mono_class_init_internal (MonoClass *klass);
```

Comment:
Should be after `MONO_BEGIN_DECLS`.

Reply:
This was deliberate. It is internal and doesn't need `extern C`.

Reply:
Many cases of `G_BEGIN_DECLS` and `MONO_BEGIN_DECLS` are suspicious and should be removed, including this one. The header is not public.
But removing them is considered excess change, esp. when we aren't yet commited to C++, and does indeed carry risk -- of needing to add back some `G_EXTERN_C` as found and needed.

Reply:
Arguably, as a replacement for a prior function, it should extern C, to maintain consistenty but that doesn't seem super well motivated. Is consistency the reason, or other?